contracts/DepositManager.sol

```diff
@@ -79,13 +79,10 @@ contract DepositManager {
             nameRegistry.getContractDetails(ParamManager.ACCOUNTS_TREE())
         );
 
-        AddCoordinatorLeaves();
-    }
-
-    function AddCoordinatorLeaves() internal {
-        // first 2 leaves belong to coordinator
-        accountsTree.appendLeaf(ZERO_BYTES32);
+        // Coordinator
         accountsTree.appendLeaf(ZERO_BYTES32);
+        // Reddit
+        accountsTree.appendLeaf(redditPubkeyHash);
```

Comment:
Register redditPubkeyHash when deploying DepositManager

Reply:
Can we use appendDataBlock here instead, so that it emits an event which the node can catch and register the pubkey?

Reply:
Sure thing